docs/_documentations/checkingstatuses.md

```diff
@@ -20,23 +20,19 @@ The application status shows the current state of your application.
 
 * Starting the application
   * When you start Codewind, it starts all the applications unless you have disabled them.
-  * To enable a project that has been disabled, visit the *Project Overview* page by using the context menu in the Codewind view and click the **Enable project** button to start the application and build containers.
+  * To enable a project that has been disabled, visit the *Project Overview* page by using the context menu in the Codewind view and click the **Enable project** button to build containers and start the application.
 * Stopping an application
   * When you stop Codewind, its applications also stop.
-  * To disable a project, navigate to the **Project Overview** page and click the **Disable project** button to stop the application and build containers to save resources.
-
-Troubleshooting the application status
-* If your application goes into the **Stopped** state unexpectedly or stays in the **Starting** state longer than expected, check the application logs to see whether something went wrong. Open the application logs from the project’s context menu. Problems with the application build or server configuration can make the application fail to start and create errors in the log files.
-* Even without errors, the **Stopped** state can occur if the [context root](project-settings.html#context-root) of the application is unreachable.
+  * To disable a project, navigate to the **Project Overview** page and click the **Disable project** button to stop the application and containers to save system resources.
 
 ## Build status
 
 The build status shows the current stage of your application in the build lifecycle.
 * When you create or import your application, the state is **Unknown**.
-* The application is then subsequently put on a waiting queue for its build to begin. The state when the application is on the waiting queue is **Queued**.
+* The project is then subsequently put in a build queue. When a project is in the build queue its build state is **Queued**.
   * The default maximum number of builds that can be run concurrently is three.
-  * You can adjust the maximum number of concurrent builds by setting the `MC_MAX_BUILDS` environmental variable.
-* If your build fails, view the build logs. The build logs can be opened from the project's context menu. In the build logs, you can view error messages that describe the type and cause of failure.
+  * If you must adjust the maximum number of concurrent builds, you can set the `MC_MAX_BUILDS` environmental variable in the `codewind-pfe` container. It is not recommended to alter this value because there can be a significant impact on performance.
```

Comment:
I think it may be better to move that to troubleshooting. Moved it to a new section in Troubleshooting called 'Adjusting the maximum number of concurrent builds'.